### PR TITLE
[sqlitepy/en] Learn SQLite with Python in Y minutes

### PR DESCRIPTION
**Pull request by Arihant25**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: sqlite.html.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+language: SQLite with Python
+filename: learnsqlite.py
+contributors:
+  - ["Arihant Tripathy", "https://github.com/Arihant25"]
+---
+
+SQLite is a relational database management system (RDBMS). It is a software library that implements a self-contained, serverless, zero-configuration, transactional SQL database engine. SQLite is the most widely used open-source database engine.
```
Comment (Menelion) [inline sqlite.html.markdown:8]:
```suggestion
[SQLite](https://sqlite.org) is a relational database management system (RDBMS). It is a software library that implements a self-contained, serverless, zero-configuration,
transactional SQL database engine. SQLite is the most widely used open-source database engine.

```

File: sqlite.html.markdown [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+cursor.execute('''
+INSERT INTO books VALUES(1, "Harry Potter", "J.K. Rowling", 9.3)
+''')
+
+# Commit the changes to the database so that they are saved.
+
+db.commit()
+```
+
+## Further Reading
```
Comment (Menelion) [inline sqlite.html.markdown:45]: Maybe show more Python module methods if they exist?

**Review by Menelion — CHANGES_REQUESTED** [review]

**Comment by imba-tjd** [comment]
Personally I would :-1:

The official docs https://docs.python.org/3/library/sqlite3.html has more and clear examples and description.

**Comment by Arihant25** [comment]
I added a link to the official docs so people can read more there

File: sqlite.html.markdown (hunk 2 above)
Comment (Arihant25) [reply sqlite.html.markdown:51]: I added the close method. I don't think any other methods are really essential for basic usage.

**Comment by Arihant25** [comment]
@Menelion So can you merge this PR now?

**Comment by bowmanjd** [comment]
A nice little write-up. But I am failing to see the need, given that there is already a Learn SQL in Y minutes, and a Learn Python in Y minutes, and very clear and simple instructions in the Python docs.

If we had an article for how to interface with SQLite (or MariaDB, Postgres, Oracle, etc.) in every language that supported it, then Learn X in Y minutes would become cluttered indeed.

**Pull request closed without merging**